### PR TITLE
Demo application

### PR DESCRIPTION
**Pull request by pvg-alex**
## Description
A demo app showing how to create and use an instance of `IronBankCache` with retrofit (simulated).

### CODE REVIEW

File: demoapp/src/main/java/com/quizup/ironbank/demoapp/DemoActivity.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    private void findViews()
+    {
+        firstEditText = (EditText) findViewById(R.id.first_edit_text);
+    }
+
+    private void getCache()
+    {
+        IronBank ironBank = new IronBank(1, getFilesDir(), new Gson());
```
Comment (pvg-mikael) [inline demoapp/src/main/java/com/quizup/ironbank/demoapp/DemoActivity.java]: ```1``` should maybe be a constant. Also add simple comments to the demo code?

**Comment by pvg-mikael** [comment]
@pvg-alex In general this look good. Though it's a good practice to write simple comments to a demo code. Not required, up to you.

**Comment by pvg-alex** [comment]
It would be nice to add some more comments, I'll think what can be added.

**Comment by pvg-mikael** [comment]
@pvg-alex On another note. Iron-bank only use RxJava observables for async operations right? Have you thought of supporting callbacks and message-buses frameworks (i.e. Otto). Would be a lot of work implementing a flux pattern just using Observables. http://fluxxor.com/what-is-flux.html

**Pull request merged**